test/helpers/azuremonitor/azure-http-data-collector-api.json

```diff
@@ -0,0 +1,258 @@
+{
```

Comment:
Maybe add a doc.go to this directory.  I assume this is a rule file for the mock.  It would be useful if you could highlight the relevent bits and how they work in providing a response

Reply:
added